lib/src/parser/recursive.dart

```diff
@@ -210,8 +218,14 @@ class RecursiveAstParser {
       references: references,
     );
     if (deSugarTemplateAst != null) {
-      return new EmbeddedTemplateAst.from(
+      TemplateAst origin = deSugarTemplateAst;
+      origin = new StarAst.from(
```

Comment:
This doesn't seem to support `template="ngFor ...` yet, is that intentional, or am I wrong?